### PR TITLE
Dév env for replication

### PR DESCRIPTION
**Pull request by bersace**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -123,6 +123,84 @@ infrastructure :
...
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      PRIMARY_HOST: postgres  # Setting PRIMARY_HOST trigger the standby mode.
+    command: postgres -c shared_preload_libraries=pg_stat_statements
+    volumes:
+      - data2:/var/lib/postgresql/data
+      - run2:/var/run/postgresql
+      - ./share/sql/pg_stat_statements-create-extension.sql:/docker-entrypoint-initdb.d/pg_stat_statements-create-extension.sql
+      - ./docker/setup-replication.sh:/docker-entrypoint-initdb.d/setup-replication.sh
+      com.dnsdock.alias: secondaire.temboard.docker
```
Comment (dlax) [inline CONTRIBUTING.md:158]: This line seems over-indented. Also, this does not work for me (after fixing indentation): which version of docker(-compose) is needed?

File: docker/setup-replication.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	host    replication     all             all            ${POSTGRES_HOST_AUTH_METHOD}
+	EOF
+	pg_ctl -D "$PGDATA" -w reload
+else
+	for j in {0..5} ; do
+		if check_primary_is_ready ; then
+			break
+		else
+			sleep $j
+		fi
```
Comment (dlax) [inline docker/setup-replication.sh:46]: Shouldn't we fail here if primary is not ready after 5 attempts?

**Review by dlax — APPROVED** [review]
LGTM; I checked that `make devenv` works with proposed `docker-compose.override.yml` but given my lack of experience in temboard devenv, I cannot go futher at the moment.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -123,6 +123,83 @@ infrastructure :
...
+    image: postgres:13-alpine  # MUST MATCH secondary service image !
+    volumes:
+    - ./docker/setup-replication.sh:/docker-entrypoint-initdb.d/setup-replication.sh
+
+  secondary:
+    image: postgres:13-alpine
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      PRIMARY_HOST: postgres  # Setting PRIMARY_HOST trigger the standby mode.
```
Comment (pgiraud) [inline CONTRIBUTING.md:151]: triggers

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -123,6 +123,83 @@ infrastructure :
...
+    volumes:
+    - data2:/var/lib/postgresql/data
+    - run2:/var/run/postgresql
+    - /var/run/docker.sock:/var/run/docker.sock
+    - .:/usr/local/src/temboard/
+    - ./agent/:/usr/local/src/temboard-agent/
+    links:
+    - secondary:secondary.dev
+    environment:
+      # Perists bash history. Eases reuse of pip install and hupper command when
```
Comment (pgiraud) [inline CONTRIBUTING.md:172]: Persists

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -123,6 +123,83 @@ infrastructure :
...
+      TEMBOARD_SSL_KEY: /usr/local/src/temboard-agent/share/temboard-agent_CHANGEME.key
+    command: sleep infinity
+```
+
+Start full environment with `make devenv`.
+
+Then, install and run agent like for primary agent:
+
+``` console
+$ docker-compose exec agent /bin/bash
```
Comment (pgiraud) [inline CONTRIBUTING.md:189]:
```suggestion
$ docker-compose exec agent2 /bin/bash
```

**Review by pgiraud — APPROVED** [review]

**Pull request merged**